strawberry/chalice/views.py

```diff
@@ -118,7 +118,7 @@ def create_response(
     ) -> Response:
         status_code = 200
 
-        if sub_response.status_code != 200:
+        if sub_response.status_code != status_code:
             status_code = sub_response.status_code
```

Comment:
Looking at this again, why don't we set the status to sub_response.status code immediately? seems like an unnecessary statement this way

Reply:
This was mostly to fix the "do not compare with magic numbers", I haven't checked the actual code. Will see if I can refactor it :)